cypress/integration/file-upload.js

```diff
@@ -6,31 +6,38 @@ describe("Oyster file upload and download", () => {
       // Goes to upload form
       cy.visit("http://0.0.0.0:3000/");
       cy.get("#upload-btn").click();
-      cy.location().should(location => {
-        expect(location.pathname).to.eq("/upload-form");
+      cy.location("pathname").should(path => {
+        expect(path).to.eq("/upload-form");
       });
 
       // Uploads image and submit
       cy.uploadFile("#upload-input", "ditto.png");
       cy.get("#start-upload-btn").click();
 
+      // Payment
+      cy.location("pathname").should(path => {
+        expect(path).to.eq("/payment-invoice");
+      });
+
       // Starts Polling
-      cy.location().should(location => {
-        expect(location.pathname).to.eq("/upload-started");
+      cy.location("pathname").should(path => {
+        expect(path).to.eq("/upload-started");
       });
 
       // Success (60s timeout)
       cy.location("pathname", { timeout: 60000 }).should(path => {
         expect(path).to.eq("/upload-complete");
       });
 
-      let handle; // TODO: Use this for webnode tests
       cy
         .get("#oyster-handle")
         .invoke("text")
-        .then(h => {
-          handle = h;
+        .then(handle => {
           expect(handle).to.not.be.null;
+
+          cy.visit("http://0.0.0.0:3000/download-form");
```

Comment:
Any chance we could make this localhost instead of 0.0.0.0?  Not sure why but I had to manually change it or else I got errors.